js_modules/dagster-ui/packages/ui-core/src/graph/SVGViewport.tsx

```diff
@@ -633,7 +633,7 @@ const SVGViewportStyles: React.CSSProperties = {
   overflow: 'hidden',
   userSelect: 'none',
   outline: 'none',
-  background: `url("data:image/svg+xml;utf8,<svg width='30px' height='30px' viewBox='0 0 80 80' xmlns='http://www.w3.org/2000/svg'><circle fill='${Colors.lineageDots()}' cx='5' cy='5' r='5' /></svg>") repeat`,
+  background: `url("data:image/svg+xml;utf8,<svg width='30px' height='30px' viewBox='0 0 80 80' xmlns='http://www.w3.org/2000/svg'><circle fill='rgba(103, 116, 138, 0.20)' cx='5' cy='5' r='5' /></svg>") repeat`,
```

Comment:
Using a CSS var on the inline SVG doesn't work, and it seemed like overkill to expose the rgba values in JS. It's simplest to just put an rgba value here, and this color looks fine in both modes.

Reply:
thats a bummer, hopefully we don't run into more cases of this.

Reply:
I didn't see any others, but yeah, kind of sucks. We might just have to take a different approach to rendering this pattern if it does need theming after all.